ec2api/api/security_group.py

```diff
@@ -229,6 +231,17 @@ def _validate_parameters(protocol, from_port, to_port):
         if (to_port < -1 or to_port > 255) and (to_port != 65535):
             raise exception.InvalidParameterValue(
                 _('ICMP code is out of range.'))
+    elif protocol in ['any']:
+        if from_port > to_port:
```

Comment:
It's not part of your change but I'm curious: for "protocol == ICMP" also we should have this same check of from_port < to_port, right?